client/src/components/Settings/styles.js

```diff
@@ -2,13 +2,15 @@ const styles = theme => ({
     parent: {
         display: 'flex',
         flexDirection: 'column',
-        alignItems: 'center',
-        paddingTop: '30px'
+        width: '60%',
+        paddingTop: '30px',
+        margin: '0 auto'
     },
     container: {
```

Comment:
I kind of think the Update Phone button should be underneath emails/texts switches. If not, maybe centered between the input box and switches.

Reply:
I adjusted this! The button is now under the input box.